### DISCUSSION TITLE
Why can't Chinese users access it?

### DISCUSSION THREAD

**Discussion by lxchinesszz** [vercel/vercel#6245] [Help]
Why can't Chinese users access it? Is it a restriction or is Vercel having a problem. I can't find the information on the Internet. So can only ask here, hope someone can help answer. Is there a solution?

**Comment by fu-sen** [comment]
No. You missed the relevant post just below.
https://github.com/vercel/vercel/discussions/6235

You may still need to deal with this
https://www.vercel-status.com/incidents/r758bhbklgfd
